Flow.Launcher.Core/Flow.Launcher.Core.csproj

```diff
@@ -54,7 +54,7 @@
   
   <ItemGroup>
     <PackageReference Include="Droplex" Version="1.7.0" />
-    <PackageReference Include="FSharp.Core" Version="8.0.301" />
+    <PackageReference Include="FSharp.Core" Version="8.0.401" />
```

Comment:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.4, search for bug label, ensuring that all results are captured by handling pagination properly. Categorise the result into bug fixes. Format the result as numbered list, and show PR number with the title.

In a separate sentence, give me the assignees usernames excluding jjw24 and sorted in case insensitive alphabetical order.